.kitchen.yml

```diff
@@ -7,105 +7,22 @@ provisioner:
 
 platforms:
   - name: ubuntu-14.04
-    attributes: 
+    attributes:
       postgresql:
         enable_pgdg_apt: true
   - name: ubuntu-12.04
-    attributes: 
+    attributes:
       postgresql:
         enable_pgdg_apt: true
-  - name: centos-6.5
-    attributes: 
+  - name: centos-6.6
+    attributes:
       postgresql:
         enable_pgdg_yum: true
 
 suites:
-  - name: master
-    driver:
-      vm_hostname: master-ubuntu-14
-      network: 
-      - ['private_network', {ip: '192.168.0.21'}]
-    run_list:
-    - recipe[pg-multi::pg_master]
-    excludes: ['ubuntu-12.04', 'centos-6.5']
-    attributes:
-      postgresql:
-        version: '9.3'
-      pg-multi:
-        master_ip: '192.168.0.21'
-        slave_ip: ['192.168.0.22']
-
-  - name: slave
-    driver:
-      vm_hostname: slave-ubuntu-14
-      network: 
-      - ['private_network', {ip: '192.168.0.22'}]
-    run_list:
-    - recipe[pg-multi::pg_slave]
-    excludes: ['ubuntu-12.04', 'centos-6.5']
-    attributes:
-      postgresql:
-        version: '9.3'
-      pg-multi: 
-        master_ip: '192.168.0.21'
-        slave_ip: ['192.168.0.22']
-
-  - name: master
-    driver:
-      vm_hostname: master-ubuntu-12
-      network: 
-      - ['private_network', {ip: '192.168.0.23'}]
-    run_list:
-    - recipe[pg-multi::pg_master]
-    excludes: ['ubuntu-14.04', 'centos-6.5']
-    attributes:
-      postgresql:
-        version: '9.3'
-      pg-multi:
-        master_ip: '192.168.0.23'
-        slave_ip: ['192.168.0.24']
-
-  - name: slave
-    driver:
-      vm_hostname: slave-ubuntu-12
-      network: 
-      - ['private_network', {ip: '192.168.0.24'}]
-    run_list:
-    - recipe[pg-multi::pg_slave]
-    excludes: ['ubuntu-14.04', 'centos-6.5']
-    attributes:
-      postgresql:
-        version: '9.3'
-      pg-multi: 
-        master_ip: '192.168.0.23'
-        slave_ip: ['192.168.0.24']
-
-  - name: master
-    driver:
-      vm_hostname: master-centos-6
-      network: 
-      - ['private_network', {ip: '192.168.0.25'}]
-    run_list:
-    - recipe[pg-multi::pg_master]
-    excludes: ['ubuntu-14.04', 'ubuntu-12.04']
-    attributes:
-      postgresql:
-        version: '9.3'
-      pg-multi:
-        master_ip: '192.168.0.25'
-        slave_ip: ['192.168.0.26']
-
-  - name: slave
-    driver:
-      vm_hostname: slave-centos-6
-      network: 
-      - ['private_network', {ip: '192.168.0.26'}]
-    run_list:
-    - recipe[pg-multi::pg_slave]
-    excludes: ['ubuntu-14.04', 'ubuntu-12.04']
+  - name: default
+    runlist:
+      - recipe[pg-multi::default]
```

Comment:
>```
runlist:
  - recipe[pg-multi::default]
```

Currently the tk suites are being generated with no run lists, all thanks to a missing `_` -- it's always the dang little things.
Correct version:
```
run_list:
  - recipe[pg-multi::default]
```